src/daemon/rpm-ostreed.socket

```diff
@@ -2,7 +2,7 @@
 ConditionKernelCommandLine=ostree
 
 [Socket]
-ListenSequentialPacket=/run/rpm-ostree/client.sock
+ListenStream=/run/rpm-ostree/client.sock
```

Comment:
We are declaring a `Sequential Packet` type socket in daemon.rs and client.rs, but declaring it as an `Stream` type in this WIP commit when adding the tokio-unix-ipc. For our usecase does it make a difference?

Reply:
I think we need to use `ListenSequentialPacket` if we're doing https://github.com/coreos/rpm-ostree/pull/3874/files#diff-bc43d479a8bbd812cda2b223dee82b4540dbd1ee93200a784301a55fc1cfe905R171